### PR TITLE
Conversant: Enable audio requests

### PR DESCRIPTION
**Pull request by johnwier**
Update yaml file to allow audio media type requests

### CODE REVIEW

File: static/bidder-info/conversant.yaml [hunk 1]
```diff
@@ -9,10 +9,12 @@ capabilities:
     mediaTypes:
       - banner
       - video
+      - audio
   site:
     mediaTypes:
       - banner
       - video
+      - audio
```
Comment (SyntaxNode) [inline static/bidder-info/conversant.yaml:17]: Please add exemplary jsons tests for the app audio and site audio.

Comment (johnwier) [reply static/bidder-info/conversant.yaml:17]: ok, done.  I should have done that before

**Comment by johnwier** [comment]
Is there anything else I'm missing that's blocking this PR?

**Comment by SyntaxNode** [comment]
@johnwier We're just backlogged with reviews. I do see specific handling for banner and video in `parseCnvrParams`. Confirming no special handling is required for audio.

**Review by bsardo — COMMENTED** [review]
This is looking good. @johnwier can you confirm that no additional audio specific changes are needed in `parseCnvrParams`? We see that there is banner specific and video specific logic there.

**Comment by johnwier** [comment]
yes, I can confirm that.

**Review by bsardo — APPROVED** [review]

**Pull request merged**